### PR TITLE
Added error messages for boundaries that do not work in chunk geometry model

### PR DESCRIPTION
**Pull request by SophieCoulson**
Addressing issue #2331 - added error messages when lat =90/-90

### CODE REVIEW

File: source/geometry_model/chunk.cc [hunk 1]
```diff
@@ -728,6 +728,10 @@ namespace aspect
 
               AssertThrow (point1[2] < point2[2],
                            ExcMessage ("Minimum latitude must be less than maximum latitude."));
+              AssertThrow (point1[2] > -0.5*numbers::PI,
+                           ExcMessage ("Minimum latitude cannot be exactly -90 degrees"));
```
Comment (tjhei) [inline source/geometry_model/chunk.cc]: maybe "... needs to be larger than -90 degrees" would be a better error message? Can you add a period at the end?

**Comment by MFraters** [comment]
Thanks for adding this, this makes it a lot more friendly :)
/run-tests

**Comment by jaustermann** [comment]
The wording has been updated. This looks good now if tester is happy.

**Comment by tjhei** [comment]
@SophieCoulson it looks like your git settings on your machine do not have the same email address as your github account (see the commit looks gray and doesn't have your profile picture?). Would you mind changing this option and editing your commit? I can show you how to do that. As a result this will be correctly credited to you.

**Comment by tjhei** [comment]
excellent. I think this is ready to be merged after the testers are happy.

**Pull request merged**